### DISCUSSION TITLE
Let's run our own QLever in a Cloud VPS

### DISCUSSION THREAD

**Discussion by dpriskorn** [WDscholia/scholia#2425] [Ideas]
Would anyone like to join me in doing that?
here is a list of questions I would like an answer to before starting:
1) how robust is it?
2) are there any open bugs which affect running it in production?
3) how many queries/min can it support on a Cloud VPS instance?
4) is there anything hindering volunteers from using it in a Cloud VPS?

**Comment by dpriskorn** [comment]
According to https://www.youtube.com/watch?v=IJHyjdoSN54
1) robust enough according to Hannah.
2) Yes, but only related to missing SPARQL features like
* filter not exists -> minus works just fine
* bound -> no good alternative?
* ?
3) unknown, we can test that once set up, it seems very fast/resource efficient
4) not that I can find

**Reply by dpriskorn** [reply]
Bound has been implemented

**Comment by fnielsen** [comment]
Where would it run?

**Reply by dpriskorn** [reply]
In the WMF Cloud VPS. It's easy to request. I can help set it up if anybody here want to join.

**Reply by fnielsen** [reply]
Isn't the problem that it needs to be a very big instance?

**Comment by WolfgangFahl** [comment]
@dpriskorn @fnielsen - currently we have http://qlever.wikidata.dbis.rwth-aachen.de/wikidata running - if we like we can update it daily. James Hare has volunteered to join in with his machine. Daniel is looking for resources in. his environment and of course the Wikimedia foundation is capable of supplying the necessary hardware footprint. Who would be the contact person there?

_Isn't the problem that it needs to be a very big instance?_
The QLever requirements are not very high any more. It used to be 128 GB RAM and a 2 TB SSD but you can do with less RAM abd a rotating disk these days. For proper operation a Ryzen 9 12 core with 128 GB RAM will work nicely. This is a setup i run at home.   

Please note that #2412 is the main issue we intend to work on here and this server / QLever discussion is only part of that effort.

#1774 also already brought up the QLever discussion
